launchflow/flows/deploy_flows.py

```diff
@@ -1730,7 +1730,7 @@ async def plan_promote_service(
     ):
         exception = exceptions.ServiceProductMismatch(
             service=service,
-            existing_product=existing_to_service.product.name,
+            existing_product=existing_to_service.product,
```

Comment:
I assumed there would be less instances lol, I'll stop spamming the github thread, thought it might be useful if there were only a handfull :slightly_smiling_face: